src/users/cd-redirect-to-login.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div class="cd-redirect-to-login">
+    <p class="cd-redirect-to-login__heading">{{ $t('Have you booked an event before?') }}</p>
+    <div>
+      <router-link :to="login" class="btn btn-secondary cd-redirect-to-login__button">{{ $t('Log into your account') }}</router-link>
+    </div>
+  </div>
+</template>
+
+<script>
+
+  export default {
+    name: 'RedirectToLogin',
+    props: ['url'],
+    computed: {
+      login() {
+        return {
+          path: `/login?next=${this.url}`,
```

Comment:
Should perhaps use the new login page? If so, should use a name-based to in router-link instead of a path.

Reply:
The login page wasn't done yet at the time, but will defo change, even tho' we need to remember to change it back later on